app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
```

Comment:
if you need to iterate objects, but not their indexes, it's better to use such construction
```suggestion
    for friend in friends:
```